gitbook-docs/evm/endpoint.md

```diff
@@ -0,0 +1,257 @@
+# Proof Market Endpoint for Ethereum
+
+Proof Market EVM Endpoint is a solution that enhances the capabilities of [Proof Market](https://proof.market) by enabling direct proof ordering for Ethereum applications.
+This extension allows for trustless, provable computation composability for all EVM applications.
+The Ethereum Endpoint acts as a gateway interface facilitating communication between EVM applications and Proof Market.
+It enables several crucial operations:
+* Requesting zkProofs: EVM applications can directly request zero-knowledge proofs for specific computations.
+* Payment Facilitation: Payment for proof producers can be facilitated directly through Ethereum, streamlining the payment process.
+* Proof Verification: The Ethereum Endpoint acts as an additional verifier in the proof generation pipeline, ensuring the validity of incoming proofs.
```

Comment:
It may be the Ethereum endpoint for us because we only have one. But for the Eth devs, so many things are Eth endpoints.

Why is it an additional verifier? I can't imagine someone doing the in-EVM verification twice because it's expensive.

```suggestion
* Proof Verification: the Proof Market Endpoint acts as an additional verifier in the proof generation pipeline, ensuring the validity of incoming proofs.
```